index.html

```diff
@@ -0,0 +1,264 @@
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    
+    <title>Capstone1</title>
+  </head>
+  <body>
+    <header>
+      <nav class="tools">
+        <img src="images/facebook-icon.png" alt="" class="fb_icon">
+        <img src="images/twitter-icon.png" alt="" class="fb_icon">
+        <a href="#" class="english">English</a>
+        <a href="#" class="english">My page</a>
+        <a href="#" class="english" id="last_tool">Logout</a>
+      </nav>
+      <nav id="NavMobile">
+        <img src="images/Hamburger.png" alt="" class="hamburger" id="hamburger1"/>
+        <img src="images/x-icon.png" alt="" class="hamburger" id="hamburger2"/>
+        <ul class="ul_nav" id="ul_nav">
+          <li id="home"><a class="links_Nav" >Home</a></li>
+          <li id="about"><a class="links_Nav" >About</a></li>
+          <li><a class="links_Nav" >Join</a></li>
+          <li><a class="links_Nav" href="#">Program</a></li>
+          <li><a class="links_Nav" href="#">Sponsor</a></li>
+          <li><a class="links_Nav" href="#">News</a></li>
+        </ul>
+        <!-- <img src="images/PngItem_1074369.png" alt="" class="world_img" /> -->
+      </nav>
+      <nav id="NavDesck">
+        <div class="logo_div_Nav" id="lodNav">
+          <img src="images/v-icon-3.jpg" alt="" class="img_logo2">
+          <h2 class="title_footer">The Vitron community</h2>
+        </div>
+        <ul id="Ul_desk">
+          <li id="home2"><a class="links_Nav" >Home</a></li>
+          <li id="about2"><a class="links_Nav" >About</a></li>
+          <li><a class="links_Nav" >Join</a></li>
+          <li><a class="links_Nav" href="#">Sponsor</a></li>
+          <li><a class="links_Nav" href="#">News</a></li>
+          <li><a class="links_Nav" id="Ignite" href="#">V Ignite</a></li>
+        </ul>
+        <!-- <img src="images/PngItem_1074369.png" alt="" class="world_img" /> -->
+      </nav>
+     
+    </header>
+
+    <section class="home_page" id="home_page">
+      <div class="section_intro">
+        <h2 class="Intro">"Hello, make it bounce with Vitron"</h2>
+        <h1 class="big_title">THE VITRON COMMUNITY SAYS HELLO.</h1>
+        <p class="p_description" id="p_desckA">
+          Vitron Is a new brand enterprise based on software and hardware
+          devellopement, It's a company wich is extended over 10 contries in the
+          world.We gonna celebrate our lancemnet of the new product in the
+          market
+          <span>The Vitron 2.1 subwoofer</span> in may 2022
+        </p>
+        <h2 class="date_conf">2022.7.20(Mon)~22(Tue)</h2>
+        <h3 class="adresse">@The Liberty Warehouse,Red Hook,Brooklyn.</h3>
```

Comment:
- [ ] Kindly adjust this part to be more to the left to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/163587391-66891bac-a4ad-4383-aec6-a060be388d29.png) | ![image](https://user-images.githubusercontent.com/24703171/163587651-c9ffa462-dcd0-45da-9a35-3fe76b995e85.png) |